server/pulp/server/webservices/views/serializers/__init__.py

```diff
@@ -279,7 +280,7 @@ def translate_criteria(self, model, crit):
             crit_dict['sort'] = sort
         if crit.fields:
             crit_dict['fields'] = [self._translate(model, field) for field in crit.fields]
-        return criteria.Criteria.from_dict(crit_dict)
+        return Criteria.from_dict(crit_dict)
```

Comment:
I think we should fix this circular import by translating the `crit_dict` and building the `Criteria` object where this function is used. This should remove the need to import Criteria in the serializer which makes more sense to me. This looks like it should be addressed separately from this issue however, so I would be satisfied by filing a new task.